Build iOS failed

**Flutter version:**
```
[✓] Flutter (Channel stable, v1.7.8+hotfix.4, on Mac OS X 10.14.5 18F203, locale en-SG)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 10.3)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.36.1)
```

**Error stacktrace:**
```dart
Running Xcode build...                                                  
Xcode build done.                                            1.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    ../.pub-cache/hosted/pub.dartlang.org/fluttie-0.4.0/ios/Classes/FluttiePlugin.m:14:16: error: expected identifier
        result([@NO]);
                   ^
    1 error generated.

Could not build the application for the simulator.
Error launching application on iPhone Xs.
```

--- a/ios/Classes/FluttiePlugin.m
+++ b/ios/Classes/FluttiePlugin.m
@@ -11,7 +11,7 @@
 
 - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   if ([@"isAvailable" isEqualToString:call.method]) {
-    result([@NO]);
+    result(@(NO));
   } else {
     result(FlutterMethodNotImplemented);
   }

